moved/src/move_execution/tests/transfer.rs

```diff
@@ -1,3 +1,5 @@
+use std::u64;
```

Comment:
nit: you can check clippy locally:
`cargo clippy --all-features --all-targets`

Reply:
I did a few times when cleaning up, but it's my nvim completion adding unneeded imports without me asking. Thanks for pointing out, still.